Isle of Trust/src/routes/TutorialGameView.tsx

```diff
@@ -596,46 +594,56 @@ class TutorialView extends React.Component<StartInfo, GameViewState> {
 
     //Determines when to render the EndOfLevel component based on the level's stageCount or turnCount
     renderEndOfLevel = () => {
-
-
         const checkResource = this.state.map.getVertices();
 
-        
         // If user runs out of resources user must restart level
-        if (checkResource[0].resources <= 0 ){
-            return <OutOfResources level={this.props.level}/>
+        if (checkResource[0].resources <= 0) {
+            return <OutOfResources level={this.props.level} />;
         }
-                                         
-        //Level 0
-        if (this.props.level === 0 && this.state.stageCount === 27) {
 
-                return <EndOfLevel level={this.props.level} 
-                success={true} 
-                mapAgents = {[]}/>;
+        //Level 1
+        if (this.props.level === 1 && this.state.stageCount === 27) {
+            return (
+                <EndOfLevel
+                    level={this.props.level}
+                    success={true}
+                    mapAgents={[]}
+                />
+            );
         }
-        //Levels 1-6
+        //Levels 2-7
         if (
-            this.props.level >= 1 &&
-            this.props.level <= 6 &&
+            this.props.level >= 2 &&
+            this.props.level <= 7 &&
             this.state.turnCount === 10
         ) {
-        // Check to make sure users can pay mortgage
-        let payMortgage = 0;
-        for (let i = 0; i <= checkResource.length-1; i++){
-            if (checkResource[i].resources >= 5){payMortgage++}
-        }
-        // All players can pay mortgage
-        if (payMortgage == checkResource.length){
-            return <EndOfLevel level={this.props.level} 
-            success = {true}
-            mapAgents = {checkResource}/>;
-        }
-        // Not all players can pay 5 ton mortgage
-        else {
-            return <EndOfLevel level={this.props.level} 
-            success = {false}
-            mapAgents = {checkResource}/>;
-        }
+            // Check to make sure users can pay mortgage
+            let payMortgage = 0;
+            for (let i = 0; i <= checkResource.length - 1; i++) {
+                if (checkResource[i].resources >= 5) {
+                    payMortgage++;
+                }
+            }
+            // All players can pay mortgage
+            if (payMortgage == checkResource.length) {
+                return (
+                    <EndOfLevel
+                        level={this.props.level}
+                        success={true}
+                        mapAgents={checkResource}
+                    />
+                );
+            }
+            // Not all players can pay 5 ton mortgage
+            else {
+                return (
+                    <EndOfLevel
+                        level={this.props.level}
+                        success={false}
+                        mapAgents={checkResource}
+                    />
```

Comment:
Change line 642 to` success={true}` to test the end of level pop up for level 7 without actually having to win the level